File: src/Client/src/models/ProjectDetails/EstimationVm.ts
export interface IEstimationVm{
    id: string
    userPositionId: string
    userPositionIdent: string
    userPositionName: string
    hours: number
}

export class EstimationVm implements IEstimationVm {
    id: string = ''
    userPositionId: string = ''
    userPositionIdent: string = ''
    userPositionName: string = ''
    hours: number = 0
}

File: src/Server/Taskly.Application/App/Projects/Queries/GetProjectDetailedInfo/EstimationVm.cs
namespace Taskly.Application.Projects
{
    public class EstimationVm
    {
        public Guid Id { get; set; }
        public Guid UserPositionId { get; set; }
        public string? UserPositionIdent { get; set; }
        public string UserPositionName { get; set; }
        public int Hours { get; set; }
    }
}